### PR TITLE
Update OneLogin.md

### PR DESCRIPTION
**Pull request by lheapcs**
Updates for additional information on how to use certain methods.

### CODE REVIEW

File: pages/connector-authentication/OneLogin.md [hunk 1]
```diff
@@ -24,3 +24,31 @@ To set up the OneLogin connector in Cyclr, go to your **Cyclr Console**.
...
 
 6. Select **Save Changes**.
+
+## Additional information
+
+The **Create App** method accepts a wide range of possible request parameters beyond what's mapped as standard in Cyclr. You can map these fields manually, or you can generate the fields based on an existing application. To generate fields:
+
+1. Go to the **Edit Connector** page for the OneLogin connector.
+2. Under the **Methods & Fields** heading, find and select the **Get App Template** method.
+3. Run this method specifying the template app which will be used to create a new app.
```
Comment (erin-lw) [inline pages/connector-authentication/OneLogin.md:34]:
```suggestion
3. Specify the template app you want to use, and run this method.
```

File: pages/connector-authentication/OneLogin.md [hunk 2]
```diff
@@ -24,3 +24,31 @@ To set up the OneLogin connector in Cyclr, go to your **Cyclr Console**.
...
+4. Copy the returned JSON under **Return Value**.
```
Comment (erin-lw) [inline pages/connector-authentication/OneLogin.md:35]:
```suggestion
4. Copy the returned JSON script under **Return Value**.
```
pretty unnecessary clarification, I admit :)

File: pages/connector-authentication/OneLogin.md [hunk 3]
```diff
@@ -24,3 +24,31 @@ To set up the OneLogin connector in Cyclr, go to your **Cyclr Console**.
...
+5. Under the **Methods & Fields** heading, find and select the **Create App** method.
+6. In the **Request Fields** section, find and select the red magnifying glass icon.
+7. Paste the copied JSON into the **Sample Data** box.
```
Comment (erin-lw) [inline pages/connector-authentication/OneLogin.md:38]:
```suggestion
7. Paste the copied JSON script into the **Sample Data** box.
```

File: pages/connector-authentication/OneLogin.md [hunk 4]
```diff
@@ -24,3 +24,31 @@ To set up the OneLogin connector in Cyclr, go to your **Cyclr Console**.
...
+8. Select **Generate**.
+
+This process maps the possible request fields so they're ready to use when you create a new application.
+
+The **List Users** and **Get User** methods can return custom attributes if the're available. To map these in the response, map a custom field to the attribute's location:
```
Comment (erin-lw) [inline pages/connector-authentication/OneLogin.md:43]: is it **list users** or **list user**? you use user in the steps :) if it's users, is the get user name correct?

**Review by erin-lw — APPROVED** [review]

File: pages/connector-authentication/OneLogin.md [hunk 5]
```diff
@@ -24,3 +24,31 @@ To set up the OneLogin connector in Cyclr, go to your **Cyclr Console**.
 5. Enter your OneLogin **sub domain**. You can find the domain in your OneLogin frontend URL just before `.onelogin.com`. For example, if your frontend URL is `https://testing-dev.onelogin.com/portal`, your **sub domain** would be `testing-dev`.
 
 6. Select **Save Changes**.
+
+## Additional information
+
```
Comment (erin-lw) [inline pages/connector-authentication/OneLogin.md:29]:
```suggestion

### Generate fields for the **Create App** method

```

File: pages/connector-authentication/OneLogin.md [hunk 6]
```diff
@@ -24,3 +24,31 @@ To set up the OneLogin connector in Cyclr, go to your **Cyclr Console**.
...
+2. Under the **Methods & Fields** heading, find and select the **Get App Template** method.
+3. Run this method specifying the template app which will be used to create a new app.
+4. Copy the returned JSON under **Return Value**.
+5. Under the **Methods & Fields** heading, find and select the **Create App** method.
+6. In the **Request Fields** section, find and select the red magnifying glass icon.
+7. Paste the copied JSON into the **Sample Data** box.
+8. Select **Generate**.
+
+This process maps the possible request fields so they're ready to use when you create a new application.
+
```
Comment (erin-lw) [inline pages/connector-authentication/OneLogin.md:42]:
```suggestion

### Map custom fields for the **List/Get Users** methods

```

File: pages/connector-authentication/OneLogin.md (hunk 4 above)
Comment (lheapcs) [reply pages/connector-authentication/OneLogin.md:43]: It should be 'List Users' and 'Get User' throughout

File: pages/connector-authentication/OneLogin.md [hunk 7]
```diff
@@ -24,3 +24,35 @@ To set up the OneLogin connector in Cyclr, go to your **Cyclr Console**.
...
+### Map custom fields for the **List/Get Users** methods
+
+The **List Users** and **Get User** methods can return custom attributes if the're available. To map these in the response, map a custom field to the attribute's location:
+
+1. Go to the **Edit Connector** page for the OneLogin connector.
+2. Under the **Methods & Fields** heading, find and select the **List Users** or **Get User** method.
+3. In the **Request Fields** section, find and select the red plus icon.
+4. For **List User**, set the field location to `[].custom_attributes.attributeName` where `attributeName` is the returned custom attribute.
```
Comment (erin-lw) [inline pages/connector-authentication/OneLogin.md:52]:
```suggestion
4. For **List Users**, set the field location to `[].custom_attributes.attributeName` where `attributeName` is the returned custom attribute.
```

File: pages/connector-authentication/OneLogin.md [hunk 8]
```diff
@@ -24,3 +24,35 @@ To set up the OneLogin connector in Cyclr, go to your **Cyclr Console**.
...
+3. Specify the template app you want to use, and run this method.
+4. Copy the returned JSON script under **Return Value**.
+5. Under the **Methods & Fields** heading, find and select the **Create App** method.
+6. In the **Request Fields** section, find and select the red magnifying glass icon.
+7. Paste the copied JSON script into the **Sample Data** box.
+8. Select **Generate**.
+
+This process maps the possible request fields so they're ready to use when you create a new application.
+
+### Map custom fields for the **List/Get Users** methods
```
Comment (erin-lw) [inline pages/connector-authentication/OneLogin.md:45]:
```suggestion
### Map custom fields for the **List Users** and **Get User** methods
```

**Review by erin-lw — APPROVED** [review]

**Pull request merged**